p/themes/Origine/origine.rtl.css

```
@@ -84,7 +84,7 @@
 html, body {
 	background-color: var(--background-color-light);
 	color: var(--font-color);
-	font-family: "OpenSans", "Cantarell", "Helvetica", "Arial", "PingFang SC", "Microsoft YaHei", sans-serif;
+	font-family: "Vazirmatn", "OpenSans", "Cantarell", "Helvetica", "Arial", "PingFang SC", "Microsoft YaHei", sans-serif;
```

The RTL files are automatically generated, so a processing directive would be better.

https://rtlcss.com/learn/getting-started/why-rtlcss/#Processing-Directives

Perhaps something like:
```suggestion
	font-family: "OpenSans", "Cantarell", "Helvetica", "Arial", "PingFang SC", "Microsoft YaHei", sans-serif /*rtl:prepend:"Vazirmatn", */;
```

---

Er, in the non-RTL file that is, don't apply these suggestions as is. ;-)